### ISSUE TITLE
The Rocketchat icon is not visible.

### ISSUE THREAD

**Issue by mmuejde** [getferdi/ferdi#1099]
The Rocketchat icon is not visible.
<img width="91" alt="Screenshot 2020-12-04 at 13 11 07" src="https://user-images.githubusercontent.com/24757679/101162402-9c17c400-3632-11eb-9274-6fe3c9d74bfa.png">

**Comment by Govi59** [comment]
Same here.
![image](https://user-images.githubusercontent.com/22171202/104579586-6ff15900-565c-11eb-9ad8-1cc5b7dbcbfd.png)

**Ferdi info**
Version: 5.5.0 (x64)
Electron: 8.2.3
Node.js: v12.13.0
Platform: linux

**System info**
Linux Mint 20 Cinnamon
Cinnamon version 4.6.7

**Comment by real-or-random** [comment]
This issue is still relevant.

**Comment by mhatvan** [comment]
I use RocketChat myself in Ferdi and the icon is definitely there, Ubuntu 21.04 user here.

@mmuejde @Govi59 @real-or-random Please try running 5.6.0-nightly.73 to see if the issue is fixed or still relevant.

**Comment by real-or-random** [comment]
Sorry, I don't think I have the time to switch to nightly to debug this. But I can report back after the 5.6.0 release.

**Comment by mhatvan** [comment]
@real-or-random Can you try again with https://github.com/getferdi/ferdi/releases/tag/v5.6.0

I use RocketChat daily and the icon is there.

**Comment by real-or-random** [comment]
Thanks, it's working now with v5.6.0.

Note: It was a little bit strange when I started v5.6.0 for the first time:

The built-in logo was shown correctly for a second but then it was invisible again.
Then I got a notice that services had been updated and I need to restart.

After the restart the built-in logo was shown again for a second but then it was replaced by another icon (that I previously had configured manually in an attempt to work around this bug). But in the service config UX, no custom icon was shown. I had to add a custom icon and remove it again. Now the built-in icon is shown correctly...
